pkgs/development/python-modules/pydantic-settings/default.nix

```diff
@@ -50,11 +50,21 @@ buildPythonPackage rec {
     export HOME=$TMPDIR
   '';
 
+  # We separate the tests from the package so that it does not depend on ruff.
+  # We do not want all of the downstream packages that depend on pydantic-settings to also depend on ruff.
```

Comment:
Please explain that ruff is a transitive dependency via pytest-examples.

Reply:
You are right. It should be better now. If you want to further change it, please tell me.